endpoints/openrtb2/auction.go

```diff
@@ -135,7 +135,7 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 		deps.analytics.LogAuctionObject(&ao)
 	}()
 
-	req, errL := deps.parseRequest(r)
+	req, impExtInfoMap, errL := deps.parseRequest(r)
```

Comment:
Nitpick: Looking at the code a bit more with the new flow, how do you feel about naming this `impExtInfoByID` to provide context of the map key? I don't feel strongly about this if you feel differently.

Reply:
We agreed with this name with Mansi earlier. I personally like `impExtInfoMap` more, but I'm open to rename it. @mansinahar what do you think?

Reply:
Both of those names sound good to me. I don't feel strongly about either. Will leave it up to you Veronika.